src/olympia/users/models.py

```diff
@@ -642,6 +642,8 @@ def fetch_names():
 RESTRICTION_TYPES = Choices(
     ('SUBMISSION', 1, 'Submission'),
     ('APPROVAL', 2, 'Approval'),
+    ('RATING', 3, 'Rating'),
+    ('RATING_MODERATE', 4, 'Rating Flag for Moderation'),
 )
```

Comment:
Should we add another type that would disallow both submissions and ratings ?

Reply:
Mmm, well, it'd complicate the model somewhat because there would no longer be a one-to-one between the restriction type and the `allow_xxx` action.  As it wasn't specified in the ticket as a requirement I'd lean towards adding this in a follow-up if needed.